pages/linux/pacman.md

```diff
@@ -14,9 +14,9 @@
 
 `pacman -Rs {{package_name}}`
 
-- Search the package database for a keyword:
+- Search the package database for a regular expression or keyword:
 
-`pacman -Ss icon theme`
+`pacman -Ss {{regexp}}`
```

Comment:
I think quote marks are useful not only for future-proofness, but also because it signals explicitly to the reader that any string can be inserted here (including spaces).